src/test/kotlin/com/openlattice/shuttle/IntegrationStreamSerializerTest.kt

```diff
@@ -0,0 +1,15 @@
+package com.openlattice.shuttle
+
+import com.kryptnostic.rhizome.hazelcast.serializers.AbstractStreamSerializerTest
+import com.openlattice.shuttle.control.Integration
+import com.openlattice.shuttle.hazelcast.serializers.IntegrationStreamSerializer
+
+class IntegrationStreamSerializerTest : AbstractStreamSerializerTest<IntegrationStreamSerializer, Integration>() {
```

Comment:
Prefer implementing `TestableSelfRegisteringStreamSerializer` in your serializer instead

Reply:
Okay will do!